src/chainparams.h

```diff
@@ -95,6 +97,13 @@ class CChainParams
 };
 
 /**
+ * Creates and returns a std::unique_ptr<CChainParams> of the chosen chain.
+ * @returns a CChainParams* of the chosen chain.
```

Comment:
In commit "Chainparams: Use a regular factory for creating chainparams"

Technically returns a unique_ptr